packages/jsx/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env", "@babel/preset-react"]
```

Comment:
I wonder if the root-level `babel.config.js` should include the react preset so it can be used in all the packages

Reply:
I think that is a great idea. I have never heard of it _hurting_ a project that doesn't have React, but let me know if I am mistaken